tiledb/sm/rest/rest_client.h

```diff
@@ -259,6 +260,33 @@ class RestClient {
   std::string rest_server_;
 
  public:
+  struct TileDBVersion {
+    TileDBVersion() = default;
+
+    TileDBVersion(std::tuple<int, int, int> version_tuple)
+        : major_(get<0>(version_tuple))
+        , minor_(get<1>(version_tuple))
+        , patch_(get<2>(version_tuple)) {
+    }
+
```

Comment:
```suggestion
```

I'm not seeing any usages of it and if there are any, they can use `std::make_from_tuple`.

Reply:
Done